products/knownprojects/sql/_procedures.sql

```diff
@@ -76,8 +76,10 @@ DECLARE
     _new_value text;
     _valid_fields text[];
 BEGIN
+    RAISE NOTICE 'Attempting to apply corrections for field % to %.%', _field, _schema, _table;
+    
     SELECT array_agg(column_name) FROM information_schema.columns
-    WHERE table_schema = 'public' AND table_name = _table INTO _valid_fields;
+    WHERE table_schema = _schema AND table_name = _table INTO _valid_fields;
```

Comment:
One day, we will have consistent applications of corrections across data products